doc/14-features.md

```diff
@@ -280,6 +280,9 @@ predominantly affects Windows paths e.g. `C:\` becomes `C:_`.
 
 The database is assumed to exist so this object will make no attempt to create it currently.
 
+If [SELinux](22-selinux.md#selinux) is enabled, it will not allow access for Icinga 2 to InfluxDB until the boolean
+`icinga2_can_connect_all` is set to true as InfluxDB is not providing its own policy.
+
```

Comment:
We could link directly to the 'Booleans' part of the SELinux documentation here

Reply:
Should I change the link or have an additional one on the word boolean? Because I already thought about the same and was not really sure what to prefer?

Reply:
I think having two links is fine, someone who uses SELinux is probably more interested in getting to a solution quickly while someone who dosen't or isn't sure will click the first, more general, one.

It would even make sense to have the link on `icinga2_can_connect_all` but I'm not sure how having a link on preformatted text works or looks.